data/sql/updates/pending_db_world/rev_1684800494817311100.sql

```diff
@@ -0,0 +1,19 @@
+--
+DELETE FROM `spell_script_names` WHERE `ScriptName`='spell_calling_korkron_or_wildhammer';
+INSERT INTO `spell_script_names` (`spell_id`, `ScriptName`) VALUES
+(38249, 'spell_calling_korkron_or_wildhammer'),
+(38119, 'spell_calling_korkron_or_wildhammer');
+
+UPDATE `creature_template` SET `ScriptName`='npc_korkron_or_wildhammer' WHERE `entry` IN (22059, 21998);
+
+UPDATE `creature_template` SET `speed_run` = 1.14286 WHERE (`entry` = 21998);
```

Comment:
Speeds are sniffed. Correct value is 2.28571

Reply:
@Gultask, this figure is high

Reply:
It is sniffed, cant be wrong 👍